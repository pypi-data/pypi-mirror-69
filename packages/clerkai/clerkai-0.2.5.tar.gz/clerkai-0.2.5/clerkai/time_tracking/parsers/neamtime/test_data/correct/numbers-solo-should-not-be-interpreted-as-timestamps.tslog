|tz:Europe/Helsinki

start 2016-05-13 17:00

2016-05-13 17:00, starting

2016-05-13 17:03, three minutes in

2016-05-13 17:04, some results

neamtime
36.3833333333

csv med fel rounding
36.53

csv med rÃ¤tt intern rounding
36.37

csv utan rounding
36.3833333333

something that looks like a timestamp
12.30

2016-05-13 17:04, still four minutes in

2016-05-13 17:06, some minutes later

paus->
