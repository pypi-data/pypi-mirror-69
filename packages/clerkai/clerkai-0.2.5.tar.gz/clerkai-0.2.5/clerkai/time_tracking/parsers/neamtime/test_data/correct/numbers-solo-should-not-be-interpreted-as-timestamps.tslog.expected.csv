Raw Ignore,Raw Source Line Numbers,Raw Source Lines Summary,Raw Session,Raw UTC Timestamp,Raw Work Date,Raw Timezone,Raw Time-annotated Source Lines Summary,Raw Hours,Raw Annotation,Raw Year,Raw Year-half,Raw Month,Raw Week,Raw Client,Raw Invoice,Raw Invoice row ordinal,Raw Invoice row item,Raw Invoice row item category,Raw Invoice row item reference,Raw Invoice hours,Ignore,Source Line Numbers,Source Lines Summary,Session,UTC Timestamp,Work Date,Timezone,Time-annotated Source Lines Summary,Hours,Annotation,Year,Year-half,Month,Week,Client,Invoice,Invoice row ordinal,Invoice row item,Invoice row item category,Invoice row item reference,Invoice hours,Original data
,, starting,start 2016-05-13 17:00,2016-05-13 14:00,2016-05-13,UTC, 0min starting,0.0,Uncategorized,,,,,,,,,,,,,,starting,start 2016-05-13 17:00,2016-05-13 14:00:00,2016-05-13 00:00:00,UTC,0min starting,0.0,Uncategorized,,,,,,,,,,,,"{'gmtTimestamp': '2016-05-13 14:00', 'category': 'Uncategorized', 'date': '2016-05-13', 'dateRaw': '\t2016-05-13 14:00', 'hours': 0.0, 'hoursRounded': 0.0, 'lineWithoutDate': ' 0min starting', 'text': ' starting', 'ts': 1463148000, 'tz': 'UTC', 'sessionMeta.session_ref': 'start 2016-05-13 17:00', 'sessionMeta.tzFirst': 'UTC'}"
,, three minutes in,start 2016-05-13 17:00,2016-05-13 14:03,2016-05-13,UTC, 3min three minutes in,0.05,Uncategorized,,,,,,,,,,,,,,three minutes in,start 2016-05-13 17:00,2016-05-13 14:03:00,2016-05-13 00:00:00,UTC,3min three minutes in,0.05,Uncategorized,,,,,,,,,,,,"{'gmtTimestamp': '2016-05-13 14:03', 'category': 'Uncategorized', 'date': '2016-05-13', 'dateRaw': '\t2016-05-13 14:03', 'hours': 0.05, 'hoursRounded': 0.05, 'lineWithoutDate': ' 3min three minutes in', 'text': ' three minutes in', 'ts': 1463148180, 'tz': 'UTC', 'sessionMeta.session_ref': 'start 2016-05-13 17:00', 'sessionMeta.tzFirst': 'UTC'}"
,," some results | neamtime | 36.3833333333 | csv med fel rounding | 36.53 | csv med rätt intern rounding | 36.37 | csv utan rounding | 36.3833333333 | something that looks like a timestamp | 12.30 | 2016-05-13 17:04, still four minutes in",start 2016-05-13 17:00,2016-05-13 14:04,2016-05-13,UTC," 1min some results | neamtime | 36.3833333333 | csv med fel rounding | 36.53 | csv med rätt intern rounding | 36.37 | csv utan rounding | 36.3833333333 | something that looks like a timestamp | 12.30 | 2016-05-13 17:04, still four minutes in",0.016666666666666666,Uncategorized,,,,,,,,,,,,,,"some results | neamtime | 36.3833333333 | csv med fel rounding | 36.53 | csv med rätt intern rounding | 36.37 | csv utan rounding | 36.3833333333 | something that looks like a timestamp | 12.30 | 2016-05-13 17:04, still four minutes in",start 2016-05-13 17:00,2016-05-13 14:04:00,2016-05-13 00:00:00,UTC,"1min some results | neamtime | 36.3833333333 | csv med fel rounding | 36.53 | csv med rätt intern rounding | 36.37 | csv utan rounding | 36.3833333333 | something that looks like a timestamp | 12.30 | 2016-05-13 17:04, still four minutes in",0.016666666666666666,Uncategorized,,,,,,,,,,,,"{'gmtTimestamp': '2016-05-13 14:04', 'category': 'Uncategorized', 'date': '2016-05-13', 'dateRaw': '\t2016-05-13 14:04', 'hours': 0.016666666666666666, 'hoursRounded': 0.02, 'lineWithoutDate': ' 1min some results | neamtime | 36.3833333333 | csv med fel rounding | 36.53 | csv med rätt intern rounding | 36.37 | csv utan rounding | 36.3833333333 | something that looks like a timestamp | 12.30 | 2016-05-13 17:04, still four minutes in', 'text': ' some results | neamtime | 36.3833333333 | csv med fel rounding | 36.53 | csv med rätt intern rounding | 36.37 | csv utan rounding | 36.3833333333 | something that looks like a timestamp | 12.30 | 2016-05-13 17:04, still four minutes in', 'ts': 1463148240, 'tz': 'UTC', 'sessionMeta.session_ref': 'start 2016-05-13 17:00', 'sessionMeta.tzFirst': 'UTC'}"
,, some minutes later,start 2016-05-13 17:00,2016-05-13 14:06,2016-05-13,UTC, 2min some minutes later,0.03333333333333333,Uncategorized,,,,,,,,,,,,,,some minutes later,start 2016-05-13 17:00,2016-05-13 14:06:00,2016-05-13 00:00:00,UTC,2min some minutes later,0.03333333333333333,Uncategorized,,,,,,,,,,,,"{'gmtTimestamp': '2016-05-13 14:06', 'category': 'Uncategorized', 'date': '2016-05-13', 'dateRaw': '\t2016-05-13 14:06', 'hours': 0.03333333333333333, 'hoursRounded': 0.03, 'lineWithoutDate': ' 2min some minutes later', 'text': ' some minutes later', 'ts': 1463148360, 'tz': 'UTC', 'sessionMeta.session_ref': 'start 2016-05-13 17:00', 'sessionMeta.tzFirst': 'UTC'}"
